### DISCUSSION TITLE
Translating a new questionnaire

### DISCUSSION THREAD

**Discussion by ampilafunsi** [globaleaks/globaleaks-whistleblowing-software#3518] [General]
Good morning, I've just created a new Globaleaks' questionnaire for my company. Questions are in Spanish (main language) but we need its translation in our second language (language you have already included in your platform).
How can I translate it? The new questions don't appear in the dropdown from Settings-> Text personalization -> Add personalized text and the proccess is not very intuitive (at least for me)
Thanks!!

**Comment by jovimuoz** [comment]
Hello,

You can translate it in the same questionnaires. If you has defined questions in Spanish, you need change language in right upper corner and select other language, now you can translate your questionnarie. We did this for spanish, valencian and english.

Best regards.
